src/scripts/components/audio/AudioPlayPause.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+
+export default props => {
+  return (
+    <div
+      className={
+        props.editing
+          ? "playpause__container playpause__container--disabled"
+          : "playpause__container"
+      }
+    >
+      {props.playing &&
+        <img
+          src="/assets/images/button-pause_audio.png"
+          className="waveform__button__pause"
+          onClick={props.handleTogglePlay}
+        />}
+      {!props.playing &&
+        <img
+          src="/assets/images/button-play_audio.png"
+          className="waveform__button__play"
+          onClick={props.handleTogglePlay}
+        />}
+    </div>
```

Comment:
Even though the two image tags are mostly the same, with the 'pause' and 'play' being the differing section, I'm glad that you wrote it in this way because it was really easy to read and understand.